src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/Extensions/UriExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static string GetAbsoluteOrUNCPath(this Uri uri!!)
             }
 
             // Absolute paths are usually encoded.
-            return WebUtility.UrlDecode(uri.AbsolutePath);
+            return uri.AbsolutePath.Contains("%") ? WebUtility.UrlDecode(uri.AbsolutePath) : uri.AbsolutePath;
```

Comment:
This logic was simply changed to match the logic in the other `UriExtensions` class.